client/src/features/common/LuxRoutes.tsx

```diff
@@ -49,7 +49,7 @@ const LuxRoutes: React.FC = () => (
 
         {/* BEGIN data/search views */}
         <Route path="/view/results/:tab" element={<ResultsPage />} />
-        <Route path="/view/*" element={<RoutingComponent />} />
+        <Route path="/view/:type/*" element={<RoutingComponent />} />
```

Comment:
We had removed `:type` from the route pattern because LUX routes should depend only on the ID of the entity - we cannot assume that "type" is a part of the ID. Besides, this change will break the cache viewer mode.

Reply:
Ahh ok I'll revert.